src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-card flat id="AR-step-4-container">
```

Comment:
We could probably rename the ID to something more generic at this point. Something as simple as `certify-container`

Reply:
~~I agree with Cam. Let's change this id to `certify-container` (similar to StaffPayment shared component).~~

Doh, id is already in use.

Reply:
Never mind. Just leave this alone. It's not ideal but it's consistent with shared StaffPayment component.